bin/license-checker

```diff
@@ -72,6 +67,17 @@ checker.init(args, function(err, json) {
         console.error('Found error');
         console.error(err);
     }
+
+    if (shouldColorizeOutput(args)) {
+        var keys = Object.keys(json);
+        keys.forEach(function(key) {
```

Comment:
This is a little bit ugly; tbh the colorisation of the key should be performed inside the formatter, however this is a pragmatic fix if you're OK with it for now :)